docs/_data/navigation-use-cases-no-children.yml

```diff
@@ -0,0 +1,17 @@
+name: use cases na children
```

Comment:
na children?

Reply:
Oh, was this supposed to be "no children"?

Reply:
You are right. It was a typo.